fluxcd/datadog_checks/fluxcd/metrics.py

```diff
@@ -1,13 +1,34 @@
 # (C) Datadog, Inc. 2023-present
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
-METRIC_MAP = {
-    "gotk_reconcile_condition": "gotk.reconcile.condition",
-    "gotk_suspend_status": "gotk.suspend.status",
-    "gotk_reconcile_duration_seconds": "gotk.reconcile.duration.seconds",
+
+
+FLUX_V1_METRICS = {
     "controller_runtime_active_workers": "controller.runtime.active.workers",
-    "controller_runtime_reconcile": "controller.runtime.reconcile",
-    "controller_runtime_reconcile_time_seconds": "controller.runtime.reconcile.time.seconds",
     "controller_runtime_max_concurrent_reconciles": "controller.runtime.max.concurrent.reconciles",
+    "controller_runtime_reconcile": "controller.runtime.reconcile",
     "controller_runtime_reconcile_errors": "controller.runtime.reconcile.errors",
+    "controller_runtime_reconcile_time_seconds": "controller.runtime.reconcile.time.seconds",
+    "gotk_reconcile_condition": "gotk.reconcile.condition",
+    "gotk_reconcile_duration_seconds": "gotk.reconcile.duration.seconds",
+    "gotk_suspend_status": "gotk.suspend.status",
+}
+assert sorted(FLUX_V1_METRICS) == list(FLUX_V1_METRICS)
+
+FLUX_V2_METRICS = {
+    "leader_election_master_status": "leader_election_master_status",
+    "process_cpu_seconds": "process.cpu_seconds",
+    "process_max_fds": "process.max_fds",
+    "process_open_fds": "process.open_fds",
+    "process_resident_memory_bytes": "process.resident_memory.bytes",
+    "process_start_time_seconds": "process.start_time",
+    "process_virtual_memory_bytes": "process.virtual_memory.bytes",
+    "process_virtual_memory_max_bytes": "process.virtual_memory.max.bytes",
+    "workqueue_adds": "workqueue.adds",
+    "workqueue_depth": "workqueue.depth",
+    "workqueue_longest_running_processor_seconds": "workqueue.longest_running_processor",
+    "workqueue_retries": "workqueue.retries",
+    "workqueue_unfinished_work_seconds": "workqueue.unfinished_work",
```

Comment:
Sorry to bring this up again but could we drop the unit in the several metrics that have it?

Reply:
Do you mean the "seconds" in `workqueue_unfinished_work_seconds`? That's the name of the incoming metric, I removed "seconds" from the dd name.

Reply:
🤦 sorry

Reply:
but you still have `bytes` in other metrics

Reply:
ah, that's true. fixed!